homeassistant/components/yale_smart_alarm/coordinator.py

```diff
@@ -144,8 +147,13 @@ def get_updates(self) -> dict[str, Any]:
             panel_info = data["PANEL INFO"]
 
         except AuthenticationError as error:
+            LOGGER.warning("AuthenticationError updating from Yale", exc_info=True)
```

Comment:
Please remove the logs. If we want to log a message we can pass the message as an argument to the raised exceptions. The coordinator will log the errors by default.